backend/inference.py

```
@@ -1,369 +1,186 @@
+        out["prediction"] = "deepfake"
+        out["confidence"] = int(round(origin_score * 100))
+        out["findingType"] = "ai_generated"
+        out["insufficientEvidence"] = False
+        out.pop("reason", None)
+        out["disputed"] = bool(face_score is not None and face_score < 0.5)
+        return out
+
+    if face_score is None:
+        if origin_score is not None and origin_score <= strong_real:
+            out["prediction"] = "real"
+            out["confidence"] = int(round((1.0 - origin_score) * 100))
```

Ruff (0.16.1)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)